### PR TITLE
Fix exercises on control flow

### LINKED ISSUE #112: Control flow module, solution of exercise 2 wrong
My solution

```python
def solution_range_of_nums(start: int, end: int) -> list[int]:
    """
    Write your solution here
    """
    if start <= end:
        sol = range(start,end+1)
    else:
        sol = range(start,end-1,-1)
    return [i for i in sol]
```

The accepted solution does not include the last 2 numbers when start > end.
For example, when start = 99, end = 20, accepted solution goes from 99 to 22.

### PR DESCRIPTION
**Pull request by edoardob90**
Fixes #112 and #113

### CODE REVIEW

File: control_flow.ipynb [hunk 1]
```diff
@@ -483,7 +483,7 @@
    },
    "outputs": [],
    "source": [
-    "%%ipytest\n",
+    "# %%ipytest\n",
```
Comment (yakutovicha) [inline control_flow.ipynb:486]: Why did you comment that line? Leftover from testing?

**Review by yakutovicha — COMMENTED** [review]
I think there is a leftover, the rest looks good 👍

File: control_flow.ipynb (hunk 1 above)
Comment (edoardob90) [reply control_flow.ipynb:486]: Yes, I was trying a few quick things and did not want to run the tests. You can make a suggestion to remove it

Comment (yakutovicha) [reply control_flow.ipynb:486]: Here we go.
```suggestion
    "%%ipytest\n",
```

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**